src/base/utils/Reaction.jl

```diff
@@ -61,7 +61,10 @@ function is_mass_balanced(model::StandardModel, rxn::Reaction)
     atom_balances = Dict{String,Float64}() # float here because stoichiometry is not Int
     for (met, stoich) in rxn.metabolites
         atoms = metabolite_formula(model, met)
-        isnothing(atoms) && continue # ignore blanks
+        if isnothing(atoms) # ignore blanks 
+            @warn "$met has no atoms associated with it."
```

Comment:
there are the switchable logging functions to report this

Reply:
is this correct?

Reply:
like the updated code